src/Command/BaseInvalidateCommand.php

```diff
@@ -24,8 +23,6 @@
  */
 abstract class BaseInvalidateCommand extends Command
 {
-    use ContainerAwareTrait;
```

Comment:
do we need to add a setContainer method and call setContainer in our services?

the recommended way in the upgrade instructions is to not rely on the container anymore - would that be possible here, can we figure out what our code does with the container and inject the specific things to the specific commands?

(that would be out of scope for this MR though. if commands still work with fetching the container from the application object we can merge like this for now)

Reply:
i created #604 for this